compiler/test/compilable/testatifti.d

```diff
@@ -0,0 +1,108 @@
+module testatifti;

```

Comment:
A note to an issue number and an explanation of what ATIFTI is here, please.

Reply:
Done.